src/nupic/bindings/engine_internal.i

```diff
@@ -124,12 +124,21 @@
 %include <nupic/engine/Region.hpp>
 %include <nupic/utils/Watcher.hpp>
 %include <nupic/engine/Spec.hpp>
+%ignore nupic::Link::connectToNetwork;
+%ignore nupic::Link::setSrcDimensions;
+%ignore nupic::Link::setDestDimensions;
+%ignore nupic::Link::initialize;
+%ignore nupic::Link::compute;
+%ignore nupic::Link::buildSplitterMap;
+%ignore nupic::Link::purgeBufferHead;
+%include <nupic/engine/Link.hpp>
```

Comment:
I wish we could specify exactly what we need rather than what we don't need. As new methods get added in Link, it's likely that update of ignored links here might be forgotten. Specifying what we need would make it explicit what we're getting out of the class.